src/borg/testsuite/platform.py

```diff
@@ -63,6 +86,7 @@ def set_acl(self, path, access=None, default=None, numeric_owner=False):
         item = {'acl_access': access, 'acl_default': default}
         acl_set(path, item, numeric_owner=numeric_owner)
 
+    @unittest.skipIf(not are_acls_working(), 'bad filesystem ACL implementation')
```

Comment:
change reason to "ACLs do not work"?

it does not need to be a bad implementation, it can also be just not implemented or mounted without acl support.